exercises/practice/accumulate/Get-Accumulation.tests.ps1

```diff
@@ -0,0 +1,23 @@
+BeforeAll {
+    . "./Get-Accumulation.ps1"
+}
+
+Describe 'Test Get-Accumulation.ps1' {
```

Comment:
Just to be safe: did you consider implementing all test cases in the exercise's canonical data https://github.com/exercism/problem-specifications/blob/main/exercises/accumulate/canonical-data.json? It's perfectly fine not to implement all if they don't make sense for this track, but I did want to give a heads-up.

Reply:
I did consider implementing all test cases. I think that will be in the next PR. For now I wanted to get a basic exercise built and accepted and merged to get a better grasp of the process of building the exercises and what to look for.

Reply:
Sure!